packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+import styles from "./CreateTeamForm.module.css";
+import * as Button from "../../Button";
+import { TextInput } from "../../TextInput/TextInput";
+import { useForm, FieldErrors, useController } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowsRotate } from "@fortawesome/free-solid-svg-icons";
+
+import { useContext } from "react";
+import { ToastContext } from "../../Toast/ToastContext";
+import {
+  faCircleCheck,
+  faCircleExclamation,
+  faOctagonExclamation,
+} from "@fortawesome/pro-solid-svg-icons";
+
+/**
+ * Form for creating a team
+ */
+export function CreateTeamForm() {
+  const useToast = () => useContext(ToastContext);
```

Comment:
Why not just `const toast = useContext(ToastContext);`?

Better yet [do something like this](https://github.com/thunderstore-io/thunderstore-ui/blob/master/apps/cyberstorm-nextjs/utils/SessionContext.tsx#L101-L109) to ensure the toast is not null so you don't have to check for it everywhere. And naturally do it once in the toast "module" and just import it here so it doesn't have to be repeated everywhere.